cli/cmd/plugin/ui/web/tanzu-ui/src/index.scss

```diff
@@ -134,6 +114,43 @@ code {
     background-position: center 10px;
 }
 
+// custom elements
+.code {
+    vertical-align: middle;
```

Comment:
Shouldn't we use the Typography and Colors from Clarity ?  https://clarity.design/storybook/core/?path=/docs/foundation-typography--page